src/schema/v2/searchCriteriaLabel.ts

```diff
@@ -117,7 +117,7 @@ export const resolveSearchCriteriaLabels = async (
   return labels.flat().filter((x) => x !== undefined) as SearchCriteriaLabel[]
 }
 
-async function getArtistLabels(artistIDs: string[], artistLoader) {
+export async function getArtistLabels(artistIDs: string[], artistLoader) {
```

Comment:
Move of the logic we wanted was already defined previously, so I just export these functions for re-use. Maybe some larger code re-org is warranted by this as  well 🤷🏽 ?